.github/workflows/build-and-test.yml

```diff
@@ -61,7 +61,7 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Install cross (non-x86_64 linux)
-        if: "matrix.target != 'x86_64-unknown-linux-gnu' && runner.os == 'Linux'"
```

Comment:
My linter complained about these quotes so I removed them. It should work the same. Let me know if you want this in a separate PR.